ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/default_igmp_snooping_enabled.schema.yml

```diff
@@ -0,0 +1,14 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  default_igmp_snooping_enabled:
+    documentation_options:
+      filename: Fabric Variables
+      table: Multicast Settings
+    description: |
+      Disable IGMP snooping at fabric level.
```

Comment:
When set to False?

Reply:
addressed in [76813b1](https://github.com/aristanetworks/ansible-avd/pull/2510/commits/76813b11d3b1383ef83bfea0e5c7781a986a2a8a)